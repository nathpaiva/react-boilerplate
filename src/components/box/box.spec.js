'use stict';

import React from 'react';
import renderer from 'react-test-renderer';

import Box from './';

describe('#Test Box Component', () => {
  it('#Should Box with success', () => {
    const tree = renderer.create(<Box title='Register' text='First time using of caffeine.' />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
