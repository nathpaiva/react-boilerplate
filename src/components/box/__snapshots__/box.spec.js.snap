// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Test Box Component #Should Box with success 1`] = `
<h1>
  Register
   and
  <small>
    First time using of caffeine.
  </small>
</h1>
`;
